types/types.go

```diff
@@ -505,6 +506,11 @@ type SystemContext struct {
 	// === dir.Transport overrides ===
 	// DirForceCompress compresses the image layers if set to true
 	DirForceCompress bool
+
+	// HTTPClient is the HTTP client to use.  If the HTTPClient field is not nil,
+	// the docker client is created with HTTPClient, and the below cert
+	// config are ignored.
```

Comment:
There are no options “below” — and while this makes sense in a way, it can lead to surprising behavior changes.